sktalk/corpus/write/writer.py

```diff
@@ -1,7 +1,27 @@
 import abc
+import json
+import os
 
 
 class Writer(abc.ABC):
     @abc.abstractmethod
-    def write(self):
+    def asdict(self):
         return NotImplemented
+
+    def write_json(self, path: str = "./file.json"):
+        """
+        Write an object to a JSON file.
+
+        Args:
+            name (str): The file name
+            directory (str): The path to the directory where the .json
+            file will be saved.
+        """
+        ext = os.path.splitext(path)[1]
+        if ext != ".json":
+            path += ".json"
```

Comment:
```suggestion
        _path = Path(path).with_suffix(".json")
```
Change `path` to ` _path` below.